src/user/user.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller, Get, Put, UseGuards, Body, Patch } from '@nestjs/common';
+import { UsersService } from './user.service';
+import { AuthGuard } from '@nestjs/passport';
+import { GetUser } from 'src/auth/decorator/getUser.decorator';
+import { User } from '@prisma/client';
+import { UpdateUserDto } from './dto/updateUser.dto';
+
+@Controller('api')
+export class UsersController {
+  constructor(private usersService: UsersService) {}
+
+  @UseGuards(AuthGuard('jwt'))
+  @Get('user')
+  getCurrentUser(@GetUser() user: User) {
+    return { user };
+  }
+
+  @UseGuards(AuthGuard('jwt'))
+  @Patch('user')
+  updateUser(
```

Comment:
```suggestion
  update(
```
đây là method của `UsersController` rồi nên ko cần duplicate hậu tố `user` nữa